ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -591,23 +594,23 @@ ompi_mtl_ofi_finalize(struct mca_mtl_base_module_t *mtl)
     opal_progress_unregister(ompi_mtl_ofi_progress_no_inline);
 
     /* Close all the OFI objects */
-    if (ret = fi_close((fid_t)ompi_mtl_ofi.ep)) {
```

Comment:
why the extra parenthesis ?

Reply:
See the commit message on https://github.com/open-mpi/ompi/pull/4494/commits/a8686a681361177da28aac6e61963600fbd30d8a.